backend/tests/config.py

```diff
@@ -129,6 +131,26 @@ def setUp(self):
         )
         self._proposal.team.append(self.user)
         db.session.add(self._proposal)
+        db.session.flush()
+
+        milestones = [
+            {
+                "title": "Milestone 1",
+                "content": "Content 1",
+                "date_estimated": 1591660203,  # random unix time in the future
+                "payout_percent": 50,
+                "immediate_payout": True
+            },
+            {
+                "title": "Milestone 2",
+                "content": "Content 2",
+                "date_estimated": 1611660203,  # random unix time in the future
```

Comment:
These tests will start to fail in 2 years. Probably best to just do some kind of `now() + 1 year` or something than give someone a headache a couple years from now.

Reply:
yeah, true. Thanks